src/Formik.tsx

```diff
@@ -31,6 +31,7 @@ export class Formik<Values = {}, ExtraProps = {}> extends React.Component<
   static defaultProps = {
     validateOnChange: true,
     validateOnBlur: true,
+    validateOnLoad: false,
```

Comment:
How about `validateOnMount`. It is not clear when "load" happens

Reply:
Agree. Btw this is on by default in v2

Reply:
Is now a good time to use v2?

Reply:
YES

Reply:
validateOnMount makes the most sense

Reply:
I am using v2.0.1-rc.12 but there is still not validateOnLoad flag. Any idea when this very important feature will be released?